pkl-core/src/test/files/LanguageSnippetTests/input/projects/project1/globbing.pkl

```diff
@@ -34,4 +34,9 @@ examples {
   ["glob-read absolute package uri"] {
     read*("package://localhost:12110/birds@0.5.0#/catalog/*.pkl")
   }
+  
+//   https://github.com/apple/pkl/issues/166
+//   ["glob-import behind local project import"] {
+//     import("@project6/children.pkl")
+//   }
```

Comment:
This is a test for #166. I suspected these changes might resolve the issue, but they do not. This change did seem to surface a clearer error though. It looks like when the `ImportGlobNode` is executed, `currentModule` has a `file:` URI while `importUri` has a `projectpackage:` URI. This disagreement causes `ModuleKeys.ProjectPackage.listElements` to eventually bark since it doesn't know about `file:` URIs.